docs/api-docs/storefront/graphql/graphql-products.md

```diff
@@ -0,0 +1,875 @@
+# Products with the GraphQL Storefront API
+
+BigCommerce's GraphQL Storefront API lets merchants on headless storefronts retrieve [products](https://support.bigcommerce.com/s/article/Product-Identifiers) powered by results from our back-end search engine. These built-in capabilities also allow Stencil developers to customize their storefronts with variant information.
```

Comment:
> powered by results from our back-end search engine

I'm actually not sure we want to mention that since it's not entirely true. `site.search.productSearch` is the one powered by a search engine - in this case, its just a regular product list